### PR TITLE
Buttons

### LINKED ISSUE #17: Request for Design Button
As a user whose needs are not met by LimbForge, I would like to submit a request for a new design based on a description and photos of my limb difference.

### PR DESCRIPTION
**Pull request by anjiro**
Changed jQuery code to use IDs instead of classes for button clicks: good practice, and we can then style all buttons the same. We also added a "Request Design" button

closes https://github.com/e-nable/LimbForge/issues/17

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -1,3 +1,4 @@
 .DS_STORE
 node_modules
 .idea*
+*~
```
Comment (kaitlynhova) [inline .gitignore]: Did you mean to add this, or was it an artifact of some sort?

File: README.md [hunk 2]
```diff
@@ -1,28 +1,42 @@
-#LimbForge
```
Comment (kaitlynhova) [inline README.md]: Can you please remove any change to the READ.ME that doesn't have anything to do with adding the button? If you want to change READ.ME stuff, feel free to make another PR with the changes and Andreas will check it out for approval.

File: .gitignore (hunk 1 above)
Comment (anjiro) [inline .gitignore]: That's to ignore emacs backup files, which end with a ~.

File: README.md (hunk 2 above)
Comment (anjiro) [inline README.md]: Hm, not sure how that ended up in there. I'll try to figure out how to drop
that again. I might do a new pull request...

**Comment by anjiro** [comment]
Canceling this pull request in favor of cleaner ones.

**Pull request closed without merging**